tests/test_all_rules.py

```diff
@@ -641,6 +641,39 @@ def test_deprecated_rules_modified(self):
         if result:
             self.fail(f"Deprecated rules {result} has been modified")
 
+    @unittest.skipIf(os.getenv('GITHUB_EVENT_NAME') == 'push',
+                     "Skipping this test when not running on pull requests.")
+    def test_rule_change_has_updated_date(self):
+        """Test to ensure modified rules have updated_date field updated."""
+
+        rules_path = get_path("rules")
+        rules_bbr_path = get_path("rules_building_block")
```

Comment:
```suggestion
        rules_path = get_path("rules")
        rules_bbr_path = get_path("rules_building_block")
```
This looks good, the rest if this comment is a NOP until this is merged.
Note for our future selves: we will need to merge this in with DAC to use the rules config. Using this comment as to anchor reference.